minos/microservice/cli.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import (
+    Path,
+)
+from typing import (
+    Optional,
+)
+
+import typer
+
+from minos.common import (
+    MinosConfig,
+)
+from minos.microservice import (
+    EntrypointLauncher,
+)
+
+app = typer.Typer()
+DEFAULT_CONF_PATH = "tests/config.yml"
```

Comment:
I would not provide a default config, at least by the moment (the issue https://github.com/Clariteia/microservice/issues/2 will treat that)